### ISSUE TITLE
Respiratory syncytial virus bronchiolitis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6660]
**Preferred term label:**
Respiratory syncytial virus bronchiolitis

**Synonyms**
RSV bronchiolitis

**Definition (free text, please give PubMed ID)**
Inflammation of the bronchioles, caused by the respiratory syncytial virus.
PMID: 28722988

**Parent term (use hpo.jax.org/app)**
Bronchiolitis

**Comment by pnrobinson** [comment]
This is better characterized as a disease
@nicolevasilevsky  can we make a Mondo term for this?
It should live somewhere here -- can we discuss with Chris.
https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0002465
